changelog.d/828.bugfix

```diff
@@ -0,0 +1 @@
+Fix docker instances not starting due to being unable to load a dynamic library.
```

Comment:
I think that changelog messages should relate to the latest stable release.
This wasn't broken in the latest stable release, but got introduced in #825.